neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -125,9 +146,12 @@ def _log_metrics(self, logs, category: str, trigger: str):
                 pass
 
     def on_train_begin(self, logs=None):  # pylint:disable=unused-argument
-        self._metric_logger["model/summary"] = _model_summary_file(self.model)
+        self._model_logger["summary"] = _model_summary_file(self.model)
         self._metric_logger["model/optimizer_config"] = self.model.optimizer.get_config()  # it is a dict
```

Comment:
Let's update it accordingly with `self._model_logger`

Reply:
Done.